SingularityUI/app/selectors/requests.es6

```diff
@@ -1,165 +0,0 @@
-import { createSelector } from 'reselect';
-import micromatch from 'micromatch';
-import fuzzy from 'fuzzy';
-import _ from 'underscore';
-
-import Utils from '../utils';
-
-const getRequestsAPI = (state) => state.api.requests;
-const getUserAPI = (state) => state.api.user;
-const getSearchFilter = (state) => state.ui.requestsPage;
-const getCurrentGroup = (state) => state.ui.dashboard.currentGroup;
-
-function findRequestIds(requests) {
-  return _.map(requests, (request) => {
-    return _.extend({}, request, {id: request.request ? request.request.id : request.requestId});
-  });
-}
-
-export const getStarredRequests = createSelector(
```

Comment:
were the rest of these methods unused I take it?

Reply:
Yeah none of these were used anywhere and I was wondering if it was duplicated by `selectors/requests/filterSelector` but now I'm not sure. It looks like it has selectors for some incomplete features, like `getStarredRequests`. I preferring removing unused code, but I can see how this could be helpful for completing some feature requests.